src/commands/CommandSample.disable

```diff
@@ -0,0 +1,30 @@
+import { Command } from "../Command";
+import { CommandInteraction } from "discord.js";
+import { SlashCommandBuilder } from "@discordjs/builders";
+
+// set the class name, the export at the bottom and the file name to your desired command name
+class CommandName extends Command {
+    constructor() {
+        super("bunny"); // the name under which the bot internally stores your command
```

Comment:
Actually this should be the same as the named set in `register`

Reply:
```suggestion
        super("bunny"); // the name under which the bot internally stores your command (should be the same as the named set in `register`
```